pkgs/applications/science/math/R/default.nix

```diff
@@ -12,11 +12,11 @@
 assert (!blas.isILP64) && (!lapack.isILP64);
 
 stdenv.mkDerivation rec {
-  name = "R-4.0.3";
+  name = "R-4.0.4";
```

Comment:
This should be switched to pname and version.

Reply:
Sure! Since I'm very new at Nix I just changed minimal stuff and tried to ensure (but not entirely certain) that it didn't break anything.

Reply:
Thats alright. Fixed it in https://github.com/NixOS/nixpkgs/pull/115041